aruco_lookup_locations:
  target_1: [2.841857, 2.669785]
  target_2: [-1.492051, 2.633798]
  target_3: [1.310094, 0.257306]
  target_4: [6.609853, 1.148859]